### PR TITLE
update eslint peerdep lock to include v8

### PR DESCRIPTION
**Pull request by zvan**
- updated peerDependencies eslint 👈 this is the thing that will allow you to upgrade your projects
- updated devDependencies to latest versions 👈 this is to have latest dependencies and to test that there is no peerdep missmatch when using latest.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,23 +30,23 @@
...
-    "typescript": "4.1.3"
+    "eslint-plugin-angular": "4.1.0",
+    "eslint-plugin-jsx-a11y": "6.5.1",
+    "eslint-plugin-react": "7.29.2",
+    "eslint-plugin-react-hooks": "4.3.0",
+    "eslint-plugin-react-native": "4.0.0",
+    "typescript": "4.6.2"
   },
   "peerDependencies": {
     "@angular-eslint/template-parser": ">=2.0.2 <13.0.0",
```
Comment (JostHren) [inline package.json:46]: We use 13.1.0 in our project, so this condition is making us some problems. Is it possible to change it? 🤔

**Review by JostHren — CHANGES_REQUESTED** [review]

**Review by ZanMarolt — APPROVED** [review]
I have tested these changes in within our project and eslint 8.0.0

No problems 👍

**Review by nkrmavnar — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (zvan) [reply package.json:46]: 🤔 So we have 2 issues 😄 

Would it work if we open a second MR to fix this?

Comment (JostHren) [reply package.json:46]: Yes we can update this in separate MR. 👍

**Review by JostHren — APPROVED** [review]

**Pull request merged**